test/cypress/dashboards/integration/multiple-datatables.cy.js

```diff
@@ -0,0 +1,76 @@
+describe('Multiple dataTables rendering', () => {
+    before(() => {
+        cy.visit('/dashboards/cypress/multiple-datatables');
+    });
+    const CONNECTOR_ID = 'data-connector';
+
+    it('Should render multiple dataTables properly.', () => {
```

Comment:
`Render multiple dataTables` is enough, is always "should" and "properly" in background.